### ISSUE TITLE
polycon expander for cv

### ISSUE THREAD

**Issue by matthleg** [bogaudio/BogaudioModules#222]
Hello and thanks again for your amazing modules: vcv doesn't exist without you!

It would be very usefull to have cv input for polycon8 and 16,
at the same time I like it because it so small so maybe an expander would be perfect.
Please think about it,
    all the best,
          Matthieu.

**Comment by Petervos2018** [comment]
Good idea.

Maybe for the time being you could use INFIX and AFFIX from stoermelder ? (range is bipolar)
https://github.com/stoermelder/vcvrack-packone/blob/v2/docs/Infix.md
https://github.com/stoermelder/vcvrack-packone/blob/v2/docs/Affix.md
https://library.vcvrack.com/?brand=stoermelder

**Comment by matthleg** [comment]
You're right, thanks a lot for the tip

**Comment by mdemanett** [comment]
Would this be different than using POLYOFF8/16 with IN unpatched?  Other than it would save a little space, without the scale knobs.

**Comment by Petervos2018** [comment]
Nah, for me it is exactly what I wanted, but the problem with bogaudio modules is, that you don't know what you've got until somebody tells you it's already there 😄

**Comment by matthleg** [comment]
the POLYOFF8/16 is great but to have 8 knobs I will not use (in a big patch) it's very confusing: I am doing improvised live performance and I need clarity in my patch... but I use it often.
